delta/sdk/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/resolvers/ResolversImplSpec.scala

```diff
@@ -602,39 +522,19 @@ class ResolversImplSpec extends CatsEffectSpec with DoobieScalaTestFixture with
           )
         ) { case (id, value) =>
           resolvers
-            .update(id, projectRef, 4, sourceWithoutId(value))
+            .update(id, projectRef, 3, sourceWithoutId(value))
             .assertRejectedEquals(ResolverIsDeprecated(id))
         }
       }
-
-      "tag it after it has been deprecated" in {
-        forAll(
-          List(
-            nxv + "in-project"    -> updatedInProjectValue,
-            nxv + "cross-project" -> updatedCrossProjectValue
-          )
-        ) { case (id, value) =>
-          resolvers.tag(id, projectRef, tag2, 4, 4).accepted shouldEqual resolverResourceFor(
-            id,
-            projectRef,
-            value,
-            sourceWithoutId(value),
-            tags = Tags(tag -> 1, tag2 -> 4),
-            rev = 5,
-            subject = bob.subject,
-            deprecated = true
-          )
-        }
-      }
     }
 
     val inProjectExpected    = resolverResourceFor(
       nxv + "in-project",
       projectRef,
       updatedInProjectValue,
       sourceWithoutId(updatedInProjectValue),
-      tags = Tags(tag -> 1, tag2 -> 4),
-      rev = 5,
+      tags = Tags.empty,
```

Comment:
We can get rid of it if we remove tags from the state too.

Reply:
Made the change